marklogic-junit5/local.properties

```diff
@@ -0,0 +1 @@
+mlPassword=admin
```

Comment:
This shouldn't be added - it should be gradle-local.properties, which should be gitignored.